sdks/python/apache_beam/runners/common.py

```diff
@@ -536,16 +546,32 @@ def __init__(self, window_fn, main_receivers, tagged_receivers):
     self.main_receivers = main_receivers
     self.tagged_receivers = tagged_receivers
 
+    # TODO: remove if block after per-element-output-counter released
+    experiments = RuntimeValueProvider.get_value('experiments', str, '')
```

Comment:
File a JIRA about profiling this before enabling, and reference here.

Reply:
Will add a TODO here in next revision